src/components/TrainingItem/TrainingItem.tsx

```diff
@@ -8,11 +9,19 @@ export interface Props {
   date: Date;
   content: string;
   status: "ready" | "done";
+  className?: string;
 }
 
-const HistoryItem = ({ imageUrl, nickname, date, content, status }: Props) => {
+const HistoryItem = ({
```

Comment:
```suggestion
const TrainingItem= ({
```
수정이 필요해 보입니다. 30 라인의 `HistroyItem` 도 동일합니다

Reply:
오 반영 했습니다. 감사합니다~